src/pages/episode/sections/header/index.js

```diff
@@ -3,6 +3,7 @@ import {StyleSheet, css} from 'aphrodite'
 import PersonGroup from '../../../../components/person-group'
 import {sortPeople, isPast} from '../../../../../shared/utils'
 import RSVPIcon from '../../../../components/rsvp-icon'
+import TweetLink from '../../../../components/tweet-link'
```

Comment:
Because this component is only going to be used on this page, you can probably just co-locate the component file in the `header` directory with this file.